pom.xml

```diff
@@ -176,4 +220,147 @@
             </url>
         </pluginRepository>
     </pluginRepositories>
+
+    <profiles>
+        <profile>
+            <id>distribution</id>
+            <properties>
+                <jreleaser.signing.mode>MEMORY</jreleaser.signing.mode>
+                <release.jfrog.active>NEVER</release.jfrog.active>
+                <release.mavencentral.active>ALWAYS</release.mavencentral.active>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>attach-javadocs</id>
+                                <goals>
+                                    <goal>javadoc</goal>
+                                    <goal>jar</goal>
+                                </goals>
+                                <configuration>
+                                    <attach>true</attach>
+                                    <quiet>true</quiet>
+                                    <verbose>true</verbose>
```

Comment:
quit and verbose? Are these not contrary?

Reply:
Different purposes

Reply:
Anyway, I will remove the flags after we complete #45